### ISSUE TITLE
"padded-blocks" false positive

### ISSUE THREAD

**Issue by feross** [eslint/eslint#2362]
I get the following spurious error:

```  t.js:6:1: Block must not be padded by blank lines.```

With the following config:

```
{
    "env": {
        "browser": false,
        "es6": true,
        "node": true,
    },

    "rules": {
        "padded-blocks": [2, "never"],
        "semi": [2, "never"]
    }
}
```

And the following code:

```js
var a = 10

if (a === 1) {
  console.log(1)
// comment
} else if (a === 2) {
  console.log(2)
}
```

The lack of semis with the comment cause this bug.

New issue in 0.19.0. Issue did not exist in 0.18.0.

**Comment by lo1tuma** [comment]
Duplicate of #2336

**Comment by feross** [comment]
Sorry about the dupe issue. Hopefully the minimal example is helpful.
